packer/resources/ubuntu-vivid.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+# this is run as root
+
+function new_section {
+  echo
+  echo $(date +"%F %T") $1
+  echo "----------------------------------------------------------------------------------------"
+}
+
+set -e
+mkdir -p /opt
+cp -R /tmp/features /opt/
+
+## Update index and install packages
+new_section "Configuring extra repositories"
+add-apt-repository "deb http://eu-west-1.ec2.archive.ubuntu.com/ubuntu/ vivid universe multiverse"
+add-apt-repository "deb http://eu-west-1.ec2.archive.ubuntu.com/ubuntu/ vivid main restricted"
+add-apt-repository "deb http://eu-west-1.ec2.archive.ubuntu.com/ubuntu/ vivid-updates universe multiverse"
+# sometimes apt-get update doesn't see the changes here, try sleeping for a moment
+sleep 1
+
+new_section "Updating package lists"
+apt-get update
+
+## Install packages
+new_section "Installing required packages"
+apt-get --yes --force-yes install \
+  git wget language-pack-en build-essential python-setuptools \
+  openjdk-8-jre-headless openjdk-8-jdk cloud-guest-utils jq \
+  ntp unzip python3-pip
```

Comment:
IIRC git is there because some installation processes use git to install. Bower and friends spring to mind?

To be honest though, I don't recall any specific example.